docs/learn/iota2.0/core-concepts/mana.md

```diff
@@ -1,2 +1,91 @@
-high-level from https://github.com/iotaledger/IOTA2.0-intro-wiki/pull/10
-use notes from https://github.com/iotaledger/devx/issues/161
+---
+description: Detailed overview of accounts, mana, staking and delegation in IOTA 2.0.
+keywords:
+  - IOTA 2.0
+  - Accounts
+  - Mana
+  - Staking
+  - Delegation
+  - UTXO
+  - BIC credits
+---
+
+# Mana, Staking and Delegation
+
+## Accounts
+
+Accounts are a critical aspect of the IOTA 2.0 ledger.
+They perform multiple roles, such as holding Mana for issuing [blocks](data-structures.md#blocks), staking tokens for
+validation,
+delegating tokens to other [validators](validators.md), and claiming rewards from staking or delegation.
+They contribute to digital autonomy by allowing users to be their block issuers.
+Accounts in IOTA 2.0 align with the W3C DID standard for digital identity,
+contributing to digital autonomy.
+
+## Mana
+
+Mana is a vital resource for interacting with the IOTA 2.0 ledger and updating the state via block creation.
+
+You can obtain mana by:
+
+- Holding IOTA tokens.
+- Purchasing it from other Mana holders
+- Earn it by participating in [consensus](consensus.md).
+
+Unlike traditional blockchains, block creation in IOTA is not reserved for a limited group, and any IOTA account holder
+can create blocks with their Mana.
+
+Mana exists in two forms: stored on the unspent outputs and allotted to chosen accounts as BIC credits.
```

Comment:
Practically it's three. Potential, Stored and Allotted :trollface: , But I guess that out of scope again

Reply:
But seeing that we mention it later in this article, maybe it should be mentioned here too?